scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram_transducer.py

```diff
@@ -95,10 +95,10 @@ class EvalBeamSearchNGramConfig:
     probs_cache_file: Optional[str] = None  # The cache file for storing the logprobs of the model
 
     # Parameters for inference
-    acoustic_batch_size: int = 128  # The batch size to calculate log probabilities
+    batch_size: int = 128  # The batch size to calculate log probabilities
```

Comment:
Here the name changes are fine. These are using unconventional names anyhow.

Reply:
Right. I'm ok with name changes everywhere apart from transcribe speech script